jobs/component_jobs.groovy

```diff
@@ -118,18 +118,60 @@ repos.each { Map repo ->
           echo ACTUAL_COMMIT="\${ACTUAL_COMMIT}" > \${WORKSPACE}/env.properties
         """.stripIndent().trim()
 
+        shell '''
+          #!/usr/bin/env bash
+          set -eo pipefail
+
+          # Commit description parser for passing along required sibling commits...
+
+          # Looks specifically for matches of '[rR]equires <repo>#<sha>',
+          # e.g., "requires builder#abc1234, Requires router#def5678"
+          reqs=`echo "${ghprbPullLongDescription}" | grep -o "[Rr]equires [-a-z]*#[a-z0-9]*" | grep -o "[-a-z]*#[a-z0-9]*"` || true
```

Comment:
Perhaps there is a `grep` flag that would be better to use, but currently, if this `grep` command does not find a match, it exits non-zero.  Since we find it acceptable to not find a match (which constitutes the normal case for PRs not requiring others) and I've got `set -eo pipefail` above, we don't want this non-zero exit code to cause the script itself to exit non-zero.